opm/material/fluidmatrixinteractions/EclEpsConfig.hpp

```diff
@@ -212,11 +204,13 @@ class EclEpsConfig
             enablePcScaling_  = fp.has_double("PCG");
         }
 
-        if (enablePcScaling_ && enableLeverettScaling_)
-            throw std::runtime_error("Capillary pressure scaling and the Leverett scaling function are "
-                                     "mutually exclusive: The deck contains the PCW/PCG property and the "
-                                     "JFUNC keyword applies to the water phase.");
-
+        if (enablePcScaling_ && enableLeverettScaling_) {
+            throw std::runtime_error {
+                "Capillary pressure scaling and the Leverett scaling function "
```

Comment:
I don't know the details here - but *maybe* the semi new functionality "prohibits" in the json config can be used to capture this at a very early stage. See [RTEMPVD](https://github.com/OPM/opm-common/blob/master/src/opm/parser/eclipse/share/keywords/000_Eclipse100/R/RTEMPVD) for an example.